utilities/taxonomyMaps.ts

```diff
@@ -2,7 +2,7 @@
 // Per PR discussion with Shea https://github.com/SU-SWS/ood-giving-campaign/pull/290#discussion_r1602386147
```

Comment:
I'll work on this after the mobile styles (fetching the datasource for the taxonomy)